packages/govuk-frontend/src/govuk/govuk-frontend-component.mjs

```diff
@@ -29,4 +29,37 @@ export class GOVUKFrontendComponent {
       throw new SupportError()
     }
   }
+
+  /**
+   *
+   * @protected
+   * @param {Schema} schema
+   * @param {{ [key: string]: number | string | object | undefined}} config
+   * @returns {boolean} Whether the config is valid
+   */
+  validateConfig(schema, config) {
+    if (schema.anyOf) {
+      // Check if at least one of the conditions in anyOf is met
+      for (const condition of schema.anyOf) {
+        const requiredProps = condition.required
+        if (requiredProps.every((prop) => prop in config && config[prop])) {
+          return true
+        }
+      }
+      return false
+    }
+    return true
```

Comment:
Are we able to reduce these 3x returns for readability?

Maybe using `.some()` for the "at least one" instead of a loop

Happy to help look